src/chrome/content/rules/zeus-software.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="zeus-software.com">
+	<target host="zeus-software.com" />
+	<target host="www.zeus-software.com" />
+
+	<rule from="^http://zeus-software\.com/" to="https://www.zeus-software.com/" />
```

Comment:
This line is superfluous: https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#snapping-redirects